clamav/cnudie.py

```diff
@@ -23,20 +28,105 @@
 
 def scan_resources(
     resource_nodes: typing.Iterable[cnudie.iter.ResourceNode],
+    clamav_version_info: clamav.model.ClamAVVersionInfo,
+    virus_definition_age_threshold: int,
```

Comment:
if passing-in maximum allowed age in days as an integer (which is okay), please suffix it w/ `_days` to "document" this (as an alternative, we might use datetime.timedelta, which would be somewhat more flexible)

instead of `age_threshold`, which not simply call this `virus_db_max_age_days`?